index.js

```diff
@@ -76,21 +35,17 @@ function startServer(db) {
               err = { message: err || 'none' }
             }
             var statusCode = err.code || 500
-            var msg = {
-              op: 'request.summary',
-              code: statusCode,
-              route: req.route.name,
-              method: req.method,
-              path: req.url,
-              err: err,
-              t: Date.now() - req.time()
-            }
-            if (statusCode >= 500) {
-              log.error(msg)
-            }
-            else {
-              log.warn(msg)
-            }
+            api.emit(
+              'failure',
+              {
+                route: req.route.name,
+                method: req.method,
+                url: req.url,
+                err: err,
+                t: Date.now() - req.time()
```

Comment:
Restore the `code` field in this event, so that we can convert it back into a well-formed `request.summary` log.  

Alternately, use mozlog and emit those logs directly from here, so that we don't have to duplicate that logic into each backend.